# Additional Readings by Topic

Justin C. Bagley
June 22, 2019

This file presents additional suggested readings, broken down by topic, that are not cited on the poster but are cited in the corresponding paper by Bagley et al. (in prep.). There is some overlap in references between topical sections.

## Burmeistera (Campanulaceae)

- Lagomarsino, L.P., Forrestel, E.J., Muchhala, N., Davis, C.C. 2017. Repeated evolution of vertebrate pollination syndromes in a recently diverged Andean plant clade. Evolution 71(8):1970–1985.
- Muchhala, N. 2006. The pollination biology of Burmeistera (Campanulaceae): Specialization and syndromes. American Journal of Botany 93:1081-1089.
- Muchhala, N. 2008. Functional significance of extensive interspecific variation in Burmeistera floral morphology: evidence from nectar bat captures in Ecuador. Biotropica 40:332-337.
- Muchhala, N., Potts, M.D. 2007. Character displacement among bat-pollinated flowers of the genus Burmeistera: analysis of mechanism, process and pattern. Proceedings of the Royal Society B 274:2731-2737.

## Targeted sequence capture

- Faircloth, B.C., McCormack, J.E., Crawford, N.G., Harvey, M.G., Brumfield, R.T., Glenn, T.C. 2012. Ultraconserved elements anchor thousands of genetic markers spanning multiple evolutionary timescales. Systematic Biology 61:717-726.

## Empirical Hyb-Seq papers

- Chau, J.H., Rahfeldt, W.A., Olmstead, R.G. 2018. Comparison of taxon-specific versus general locus sets for targeted sequence capture in plant phylogenomics. Applications in Plant Sciences 6(3): e1032.
- Crowl, A.A., Myers, C., Cellinese, N. 2017. Embracing discordance: phylogenomic analyses provide evidence for allopolyploidy leading to cryptic diversity in a Mediterranean Campanula (Campanulaceae) clade. Evolution 71(4):913–922.
- Kates, H.R., Johnson, M.G., Gardner, E.M., Zerega, N.J., Wickett, N. J. 2018. Allele phasing has minimal impact on phylogenetic reconstruction from targeted nuclear gene sequences in a case study of Artocarpus. American Journal of Botany, 105(3):404-416.
- Medina, R., Johnson, M., Liu, Y., Wilding, N., Heddersong, T.A., Wickett, N., Goffineta, B. 2018. Evolutionary dynamism in bryophytes: phylogenomic inferences confirm rapid radiation in the moss family Funariaceae. Molecular Phylogenetics and Evolution 120(2018):240–247.
- Stubbs, R.L., Folk, R.A., Xiang, C.-L., Soltis, D.E., Cellinese, N. 2018. Pseudo-parallel patterns of disjunctions in an Arctic-alpine plant lineage. Molecular Phylogenetics and Evolution 123:88–100.
- Vatanparast, M., Powell, A., Doyle, J.J., Egan, A.N. 2018. Targeting legume loci: a comparison of three methods for target enrichment bait design in Leguminosae phylogenomics. Applications in Plant Sciences 6(3): e1036.

## Phylogenetic methods and tools used in the paper

Papers on phylogenetic methods (theory & practice) that accommodate large-scale phylogenomic data, as used in our paper:

- Dornburg, A., Fisk, J.N., Tamagnan, J., and Townsend, J. (2016) PhyInformR: Rapid Calculation of Phylogenetic Information Content. R package version 1.0. Available at: <https://CRAN.R-project.org/package=PhyInformR>.
- Zhang, C., Sayyari, E., Mirarab, S. 2017. ASTRAL-III: increased scalability and impacts of contracting low support branches. RECOMB International Workshop on Comparative Genomics. Springer, 53-75.
[others.]

Software tools

<!-- - Bagley, J.C. 2019. PIrANHA v0.3a1. GitHub repository, Available at: <https://github.com/justincbagley/piranha>. -->
- Danecek, P., Auton, A., Abecasis, G., Albers, C.A., Banks, E., DePristo, M.A., Handsaker, R.E., Lunter, G., Marth, G.T., Sherry, S.T., & McVean, G. (2011). The variant call format and VCFtools. Bioinformatics, 27, 2156–2158.
- Dornburg, A., Fisk, J.N., Tamagnan, J., and Townsend, J. (2016) PhyInformR: Rapid Calculation of Phylogenetic Information Content. R package version 1.0. Available at: <https://CRAN.R-project.org/package=PhyInformR>.
- Hillis, D. M., Heath, T. A., & John, K. S. (2005). Analysis and visualization of tree space. Systematic biology, 54(3), 471-482.
- Junier, T., & Zdobnov, E. M. (2010). The Newick utilities: high-throughput phylogenetic tree processing in the UNIX shell. Bioinformatics, 26(13), 1669-1670.
- Maddison, W. P. and D.R. Maddison. 2018. Mesquite: a modular system for evolutionary analysis.  Version 3.51  http://www.mesquiteproject.org
[others.]

## Related phylogenomics studies cited in the paper

- Prum, R.O., Berv, J.S., Dornburg, A., Field, D.J., Townsend, J.P., Lemmon, E.M., et al. 2015. A comprehensive phylogeny of birds (Aves) using targeted next-generation DNA sequencing. Nature 526:569–573.
[others.]

## Studies providing code modified for analyses in the paper

- Prum, R.O., Berv, J.S., Dornburg, A., Field, D.J., Townsend, J.P., Lemmon, E.M., et al. 2015. A comprehensive phylogeny of birds (Aves) using targeted next-generation DNA sequencing. Nature 526:569–573.


## References

Bagley, J.C., Uribe-Convers, S., Carlsen, M., Muchhala, N. (in prep.) Assessing the utility of Hyb-Seq to infer phylogeny and <!--genomic signatures of -->introgression  in rapid angiosperm radiations: a pilot study in Neotropical Burmeistera bellflowers.
